Utilities/DataFlow/include/DataFlow/PayloadMerger.h

```diff
@@ -0,0 +1,123 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
```

Comment:
for the record: we agreed to postpone this for the moment because it would break the PR code check.